test/Polar/hessian.jl

```diff
@@ -70,8 +73,8 @@ const PS = PowerSystem
         ExaPF.∂cost(polar, ∂obj, cache)
         ∇fₓ = ∂obj.∇fₓ
         ∇fᵤ = ∂obj.∇fᵤ
-        λ  = -(∇gₓ') \ ∇fₓ
-        grad_adjoint = ∇fᵤ + ∇gᵤ' * λ
+        λ  = -(Array(∇gₓ')) \ Array(∇fₓ)
+        grad_adjoint = Array(∇fᵤ) + Array(∇gᵤ)' * λ
```

Comment:
My bad ... I should improve the tests, so it would be easier to port them on the GPU...

Reply:
I think I solved it better than the above now :-) .